core/primitives/impl/scale_codec_impl.hpp

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_PRIMITIVES_SCALE_CODEC_IMPL_HPP
+#define KAGOME_PRIMITIVES_SCALE_CODEC_IMPL_HPP
+
+#include "primitives/scale_codec.hpp"
+#include "scale/types.hpp"
+
+using namespace kagome::common::scale;
+
+namespace kagome::primitives {
+
+  // clang-format off
```

Comment:
Why?

Reply:
long function names are split in ugly way. in my opinion they shouldn't be split.
an issue to discuss at meeting.